unit_tests/schema_tests.json

```diff
@@ -0,0 +1,579 @@
+        {
+          "payload": "0723 424340",
+          "expected": false,
+          "description": "starts with 07 but has a space"
+        }
+      ]
+    },
+    "address":{
+      "pattern_path": "/definitions/PrivateIndividualName/properties/address",
+      "payload_tests": [
+        {
+          "payload": "",
+          "expected": false,
+          "description": "must not be blank"
+        },
+        {
+          "payload": "PL21 0AQ",
+          "expected": true,
+          "description": "valid post code"
+        },
+        {
+          "payload": "PL21 0CC",
+          "expected": false,
+          "description": "invalid end characters"
+        },
+        {
+          "payload": "PL 0AQ",
+          "expected": false,
+          "description": "invalid start of code"
+        },
+        {
+          "payload": "PL21 0KK",
+          "expected": false,
+          "description": "invalid end characters"
+        },
+        {
+          "payload": "PL210AQ",
+          "expected": false,
+          "description": "must have a space"
+        },
+        {
+          "payload": "PL21 ABC",
+          "expected": false,
+          "description": "Second part must start with a number"
+        }
+      ]
+    },
+    "md_ref":{
+      "pattern_path": "/definitions/md_ref",
+      "payload_tests": [
+        {
+          "payload": "",
+          "expected": false,
+          "description": "must not be blank"
+        },
+        {
+          "payload": "e-MD12345",
+          "expected": true,
+          "description": "valid md ref"
+        },
+        {
+          "payload": "x-XX12345",
+          "expected": false,
+          "description": "must start with e-MD"
+        },
+        {
+          "payload": "e-MD1234",
+          "expected": false,
+          "description": "must have 5 digits"
+        },
+        {
+          "payload": "e-MD12 45",
+          "expected": false,
+          "description": "cannot have spaces"
+        }
+      ]
+    },
+    "title_number":{
+      "pattern_path": "/definitions/title_number",
+      "payload_tests": [
+        {
+          "payload": "",
+          "expected": false,
+          "description": "must not be blank"
+        },
+        {
+          "payload": "DN1234",
+          "expected": true,
+          "description": "valid title number"
+        },
+        {
+          "payload": "12345",
+          "expected": true,
+          "description": "option start characters must start with a 1"
+        },
+        {
+          "payload": "022222",
+          "expected": false,
+          "description": "must start with a 1-9 after initial chars"
+        },
+        {
+          "payload": "DN02222",
+          "expected": false,
+          "description": "must start with a 1-9 after initial chars"
+        },
+        {
+          "payload": "DN02222",
+          "expected": false,
+          "description": "must start with a 1-9 after initial chars"
+        },
```

Comment:
This seems like a duplicate of the one directly above it unless I'm completely missing something here.